plugins/lookup/tss.py

```diff
@@ -218,6 +230,11 @@ def _get_authorizer(**server_parameters):
                 server_parameters["token_path_uri"],
             )
 
+        if server_parameters.get("token"):
```

Comment:
I think that if a value for `token` is provided then the token should take priority.
So the order of `domain` and `token` should be switched here:
```python
if server_paramters.get("token"):
...

if server_parameters.get("domain"):
...
```

Reply:
That's true.

Reply:
Fixed in 10e9cc8